### PR TITLE
Enh/modalities

### PR DESCRIPTION
**Pull request by mochic**
Adds modalities.

### CODE REVIEW

File: src/np_codeocean/upload.py [hunk 1]
```diff
@@ -336,7 +345,8 @@ def create_codeocean_upload(session: str | int | np_session.Session,
         session = np_session.Session(session)
         root = np_session.NPEXP_PATH / 'codeocean' / session.folder
         behavior = np_config.normalize_path(root / 'behavior')
-        behavior_videos = behavior.with_name('behavior-videos')
+        if modality in ('ecephys', ):
```
Comment (bjhardcastle) [inline src/np_codeocean/upload.py]: why only for ecephys? surely it should be for both

File: src/np_codeocean/upload.py [hunk 2]
```diff
@@ -355,19 +366,50 @@ def create_codeocean_upload(session: str | int | np_session.Session,
...
+                np_config.normalize_path(
+                    pathlib.Path(CONFIG["rig_metadata_dir"])
+                ),
+            )
+        except Exception:
+            logger.error(
+                "Failed to update session and rig metadata for Code Ocean upload.",
+                exc_info=True,
+            )
+    else:
```
Comment (bjhardcastle) [inline src/np_codeocean/upload.py:415]: this should be `elif session.modality == 'behavior': `

`else:` should raise a ValueError for an unexpected modality name

**Review by bjhardcastle — COMMENTED** [review]
`modality` isn't being added to the csv in your changes..

I looked at the code and it's actually called `platform`
https://github.com/AllenInstitute/np_codeocean/blob/fdacf19f6d4f61a741c69d55fb76510a8f4ad358/src/np_codeocean/upload.py#L201

File: src/np_codeocean/upload.py (hunk 1 above)
Comment (mochic) [reply src/np_codeocean/upload.py]: Addressed here : 2db595654b38665d5297fd4d648d8af9082ccb38

**Comment by mochic** [comment]
The csv stuff isnt being resolved here?:
https://github.com/AllenInstitute/np_codeocean/blob/2db595654b38665d5297fd4d648d8af9082ccb38/src/np_codeocean/upload.py#L208

File: src/np_codeocean/upload.py (hunk 2 above)
Comment (mochic) [reply src/np_codeocean/upload.py:415]: Addressed here: 776bd7d5adb5c8fc276d793fbce4f9b08034df29

**Pull request merged**